[2022-06-26 20:48:20] request.INFO: Matched route "admin_system_information_check_files". {"route":"admin_system_information_check_files","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Configure\\AdvancedParameters\\SystemInformationController::displayCheckFilesAction","_legacy_controller":"AdminInformation","_legacy_link":"AdminInformation:checkFiles","_route":"admin_system_information_check_files"},"request_uri":"https://localhost/prestashop/admin344zfkxyz/index.php/configure/advanced/system-information/files?_token=NP2VWJohusp-63Uc1QIZmRnWR8iqB_cu3maRSsZO14w","method":"POST"} []
[2022-06-26 20:48:20] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-06-26 20:48:20] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"sichanapranata@gmail.com"} []
[2022-06-26 20:50:19] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\xampp\htdocs\prestashop\src\Adapter\Requirement\CheckMissingOrUpdatedFiles.php line 70 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\xampp\\htdocs\\prestashop\\src\\Adapter\\Requirement\\CheckMissingOrUpdatedFiles.php:70)"} []
[2022-06-26 20:50:20] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-06-26 20:48:20] request.INFO: Matched route "admin_common_notifications". {"route":"admin_common_notifications","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\CommonController::notificationsAction","_legacy_controller":"AdminCommon","_legacy_link":"AdminCommon","_route":"admin_common_notifications"},"request_uri":"https://localhost/prestashop/admin344zfkxyz/index.php/common/notifications?_token=NP2VWJohusp-63Uc1QIZmRnWR8iqB_cu3maRSsZO14w&rand=1656251299877","method":"POST"} []
[2022-06-26 20:50:20] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\xampp\htdocs\prestashop\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\Handler\StrictSessionHandler.php line 48 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\xampp\\htdocs\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\StrictSessionHandler.php:48)"} []
[2022-06-26 20:50:20] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Failed to start the session: already started by PHP."). at C:\xampp\htdocs\prestashop\src\PrestaShopBundle\Resources\views\Admin\Exception\error.html.twig line 60) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to start the session: already started by PHP.\"). at C:\\xampp\\htdocs\\prestashop\\src\\PrestaShopBundle\\Resources\\views\\Admin\\Exception\\error.html.twig:60, RuntimeException(code: 0): Failed to start the session: already started by PHP. at C:\\xampp\\htdocs\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:146)"} []
[2022-06-26 20:50:20] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Maximum execution time of 120 seconds exceeded" at C:\xampp\htdocs\prestashop\vendor\symfony\symfony\src\Symfony\Component\HttpFoundation\Session\Storage\Handler\StrictSessionHandler.php line 48 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Maximum execution time of 120 seconds exceeded at C:\\xampp\\htdocs\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\StrictSessionHandler.php:48)"} []
[2022-06-26 20:50:20] request.CRITICAL: Exception thrown when handling an exception (Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Failed to start the session: already started by PHP."). at C:\xampp\htdocs\prestashop\src\PrestaShopBundle\Resources\views\Admin\Exception\error.html.twig line 60) {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to start the session: already started by PHP.\"). at C:\\xampp\\htdocs\\prestashop\\src\\PrestaShopBundle\\Resources\\views\\Admin\\Exception\\error.html.twig:60, RuntimeException(code: 0): Failed to start the session: already started by PHP. at C:\\xampp\\htdocs\\prestashop\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpFoundation\\Session\\Storage\\NativeSessionStorage.php:146)"} []
